pkg/network/ebpf/c/protocols/protocol-classification-helpers.h

```diff
@@ -236,6 +285,8 @@ static __always_inline void classify_protocol(protocol_t *protocol, const char *
         *protocol = PROTOCOL_AMQP;
     } else if (is_redis(buf, size)) {
         *protocol = PROTOCOL_REDIS;
+    } else if (is_mongo(tup, buf, size)) {
```

Comment:
do we really must use connection tuple as an identifier for mongo classification?

If possible we better to stick to the same interface for all protocols and drop that additional parameter you added to `classify_protocol` function

Reply:
we must correlate between request and response as response classification is very "soft"